app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class CategoriesController < ApplicationController
```

Comment:
Phần tạo dữ liệu này em cho vào trong namespace `admin`, lúc đó mình sẽ truy cập như thế này `http://localhost:3000/admin/categories`
Em nên đọc kỹ routes namespace ở đây https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing
rồi áp dụng vào PR này 
Tương tự như vậy cho phần tạo lesson, word

Reply:
dạ